src/Pages/Sponsors/Sponsors.jsx

```diff
@@ -1,5 +1,203 @@
-const Sponsors = () => {
-  return <div></div>;
+// SponsporsPage
+import React from "react";
+import "./sponsors.scss";
+const SponsorPage = () => {
+  return (
+    <div>
+      <body>
+        <div className="first">
+          <div className="heading">
+            <img
+              src="https://res.cloudinary.com/dex25uzgp/image/upload/v1711041756/OBJECTS_y61orj.png"
+              alt=""
+            />
+          </div>
+          <div className="sponsorContainer">
+            <div className="sponsor">
+              <img
+                src="https://res.cloudinary.com/dex25uzgp/image/upload/v1711098244/Hyundai_Logo_zqyig4.png"
+                alt=""
+              />
+            </div>
+            <div className="sponsor">
+              <img
+                src="https://res.cloudinary.com/dex25uzgp/image/upload/v1711098243/Pran_Logo_qkebp4.png"
+                alt=""
+              />
+            </div>
+            <div className="sponsor">
+              <img
+                src="https://res.cloudinary.com/dex25uzgp/image/upload/v1711098241/Oil_India-Logo_zwjxl7.png"
+                alt=""
+              />
+            </div>
+            <div className="sponsor">
+              <img
+                src="https://res.cloudinary.com/dex25uzgp/image/upload/v1711098241/NRL_Logo_sja6jq.png"
+                alt=""
+              />
+            </div>
+            <div className="sponsor">
+              <img
+                src="https://res.cloudinary.com/dex25uzgp/image/upload/v1711099239/Indian_Oil_Logo1_gx89of.png"
+                alt=""
+              />
+            </div>
+          </div>
+          <div className="bihu">
+            <img
+              src="https://res.cloudinary.com/dex25uzgp/image/upload/v1711041851/magh_bihu_1_2_edzkl9.png"
+              alt=""
+            />
+          </div>
+        </div>
+        <div>
+          <div className="heading">
+            <img
```

Comment:
Do not make the heading a complete image itself. At least the writing part, and also the Japis could be made rotatable